include/api/CAnomalyJobConfig.h

```diff
@@ -41,7 +42,8 @@ class API_EXPORT CAnomalyJobConfig {
         public:
             CDetectorConfig() {}
 
-            void parse(const rapidjson::Value& json);
+            void parse(const rapidjson::Value& detectorConfig,
+                       const CFieldConfig::TStrPatternSetUMap& ruleFilters);
```

Comment:
It would be better to make this `CDetectionRulesJsonParser::TStrPatternSetUMap` - it's exactly the same type but avoids mentioning the legacy class `CFieldConfig` in this file.